api/events/events.go

```diff
@@ -63,5 +63,6 @@ func (e *Events) Mount(root *mux.Router, pathPrefix string) {
 
 	sub.Path("").
 		Methods(http.MethodPost).
-		HandlerFunc(utils.MetricsWrapHandlerFunc(pathPrefix, "events_filter", e.handleFilter))
+		Name("events_filter").
```

Comment:
Would be good to keep the name as close as possible to the URL and avoid the interpretation step.
Ie. `curl .../logs/event` -> `logs_get_event`